spec/messager_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe Messager do
+  subject(:messager){described_class.new}
+
+  describe '#send_sms' do
+
+    it 'sends a payment confirmation text message' do
+      expect(subject).to receive(:send_sms)
+      subject.send_sms
+    end
+
+    it 'returns a confirmation as an output' do
+      expect(subject).to receive(:send_sms).and_return('Your order has been confirmed')
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)